_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "JavaLand 2017 - Welcome to Phantasialand"
+date: "2017-04-06 15:16:17"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin, Bastian"]
+---
+
+The [JavaLand 2017](https://www.javaland.eu/en/javaland-2017/) conference took place for the 4th time in the famous [Phantasialand](http://www.phantasialand.de/en/) holiday park in Brühl, near Cologne.
+The program consisted of [talks](https://programm.javaland.eu/2017/schedule.html) on the frist two days and [in-depth workshops](https://www.javaland.eu/de/programm/schulungstag) at the last and third day.
+
+Here are some random facts so that you can imagine the dimension of the largest community conference for Java in Europe:
+
+- 1600 visitors, 14 different locations in the holiday park
+- 11 parallel talk tracks, 8 session slots each day
+- 8 in-depth workshops which over 8 hours each
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://twitter.com/hashtag/welcome?src=hash">#welcome</a> to <a href="https://twitter.com/hashtag/JavaLand?src=hash">#JavaLand</a> 💙 <a href="https://t.co/wX37hzFVXM">pic.twitter.com/wX37hzFVXM</a></p>&mdash; JavaLandConf (@JavaLandConf) <a href="https://twitter.com/JavaLandConf/status/846628341209878528">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Talks
```

Comment:
please delete this headline and instead put "Talk: xy" in every headline in which you describe a talk and "Workshop: xy" in every headline you describe a workshop